README.md

```diff
@@ -36,249 +30,111 @@ require "trailblazer/test/deprecation/operation/assertions"
 include Trailblazer::Test::Deprecation::Operation::Assertions # in your test class
 ```
 
-To be able to test an operation we need 3 auxiliary methods which have to be defined at the start of your tests:
-* `default_params` (**required**): hash of params which will be always passed to the operation unless overriden by `params` or `ctx`
-* `expected_attrs` (**required**): hash always used to assert model attributes
-* `default_options` (**required if using `ctx`**): hash of options which will be always passed to the operation unless overriden by `ctx`
-
-We are also providing 2 helper methods:
-* `params(new_params)`
-* `ctx(new_params, options)`
-
-Those will merge params and options for you and return the final inputs which then can be passed to the operation under testing.
-
-Pass `deep_merge: false` to the helper methods to disable the default deep merging of params and options.
-
-*Same API for Trailblazer v2.0 and v2.1.*
```

Comment:
feels like this stuff should be explained somewhere if that still there really...I haven't been following this gem much...

Reply:
Yep, this is also moved.